src/xdocs/config_naming.xml

```diff
@@ -2143,7 +2143,8 @@ public boolean equals(Object o) {
             <tr>
               <td>accessModifiers</td>
               <td>Access modifiers of methods where parameters are checked.</td>
-              <td><a href="property_types.html#access_modifiers">Access Modifier Set</a></td>
+              <td><a href="property_types.html#AccessModifierOption[]">AccessModifierOption[]</a>
```

Comment:
actual link is `checkstyle/target/site/property_types.html#AccessModifierOption.5B.5D`

Reply:
My bad, done.